app/assets/javascripts/DropDownMenu/DropDownMenu.js

```diff
@@ -76,3 +76,7 @@ DropDownMenu.prototype.getRelatedTarget = function(event) {
   }
   return null;
 }
+
+function respondToClick() { // function that reloads the DOM for MathJax (http://www.mathjax.org/docs/1.1/typeset.html)
```

Comment:
Is it just me, or is this an odd place for this function? I wouldn't mind creating a new file for MathJaX-related functions, even if it's just the one.